lib/cli/src/commands/create_exe.rs

```diff
@@ -300,6 +318,52 @@ impl CreateExe {
     }
 }
 
+fn get_config(
```

Comment:
This function is duplicating logic that should not be needed

Reply:
It is needed, but I could move it somewhere else so it's reusable.

Reply:
Removed in https://github.com/wasmerio/wasmer/pull/3483/commits/c61a5d7cf5891bcd3e8e2ea6c5535d3ebec2eeb5